### ISSUE TITLE
Create new Select Payment screen

### ISSUE THREAD

**Issue by jh2oman** [valora-inc/wallet#2196]
Create a new Select Payment screen!

![Screen Shot 2022-04-11 at 3.43.32 PM.png](https://images.zenhubusercontent.com/6111ac24d41f1f9b126e0988/8ee88111-f7ad-4eb2-92d1-15a0edb44186)

[Figma](https://www.figma.com/file/bwIY7peeykI8K4wTrXs4iN/Fiat-Connect?node-id=940%3A17477)

AC:
* Should be able to display all existing providers, split out into categories.
* Should hook into the Add/Withdraw funds flow replacing the old Select payments screen
* Should also include removing the provider type selection from earlier in the add/withdraw flow since all types are now displayed here.
* Includes the 'Crypto Exchange' Method on the bottom of the Select Payment Screen. Takes you to a screen with the list of exchanges (same as the existing flow, just the button is in Select Payment instead of the begining of the add/withdraw flow)
* 'Select Country' is NOT in scope
* If there is only one provider, no dropdown displayed.

**Comment by cajubelt** [comment]
I vote no on the experiment thing. The main purpose here is to enable many providers with one API (thru FiatConnect), not to optimize on some UX metric. It's hard to imagine a scenario where we end up going back to the old UI.

**Comment by cajubelt** [comment]
is displaying Finclusive in scope for this? (I imagine it's not...)

**Comment by lvsuttie** [comment]
How is the list of providers ordered, if more than one for each payment type?
How do we assign "fastest" type tags to each provider?

**Comment by lisaqyli** [comment]
btw I agree that this doesn't need to be an experiment but I felt it needs to be a gated behind a remote config because it's a big new feature. I'm curious to see wdyt @jh2oman ?

**Comment by cajubelt** [comment]
just saw your comment about adding a remote config @lisaqyli -- I think that's fine, as long as it it is practical to maintain in parallel with the old UI for some short time (and doesn't break from the existing flow too much). But I don't think we should make a *big* effort to keep it roll-back-able if it's not convenient, would rather we just QA it effectively in that case.

**Comment by cajubelt** [comment]
estimated 2 points rolling over into next sprint
